### PR TITLE
Feature/tech 1656

### PR DESCRIPTION
**Pull request by Traxmaxx**
## Solution

Introduce feedback widget at Komiser dashboard

## Changes Made

- Add feedback widget
- Allow user to report an issue by capturing current screen or uploading a file
- Allow user to share their feedback

## How to Test

Check out branch and define `DISCORD_FEEDBACK_URL` when starting the backend wich the discord hook URI. Then start the frontend and click on "Leave feedback" at the top right.

## Video

https://github.com/tailwarden/komiser/assets/320272/db2ce490-42d3-46e2-90e9-f955beeb20cb

## Screenshots

Screenshot 2023-10-10 at 17 31 46
Screenshot 2023-10-10 at 17 31 57
Screenshot 2023-10-10 at 17 33 20
Screenshot 2023-10-10 at 17 33 23
Untitled

## Notes
Feedback type is still to be added

## Checklist

- [x] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [x] Changes have been thoroughly tested
- [x] <a href="https://github.com/tailwarden/docs.komiser.io">Documentation</a> has been updated, if necessary
- [x] Any dependencies have been added to the project, if necessary

## Reviewers

@[username of the reviewer]

### CODE REVIEW

File: dashboard/services/settingsService.ts [hunk 1]
```diff
@@ -458,6 +458,19 @@ const settingsService = {
     } catch (error) {
       return Error;
     }
+  },
+
+  async sendFeedback(payload: FormData) {
+    try {
+      const res = await fetch(`${BASE_URL}/feedback`, {
```
Comment (Kolawole99) [inline dashboard/services/settingsService.ts:465]: Just a quick ask, does this EP already exist on the backend

**Review by Kolawole99 — APPROVED** [review]
LGTM pending the comment I left on the endpoint availability

File: dashboard/services/settingsService.ts (hunk 1 above)
Comment (Azanul) [reply dashboard/services/settingsService.ts:465]: https://github.com/tailwarden/komiser/blob/476b273a97bce35c773f57a9c6a05f9699899e8b/internal/api/v1/endpoints.go#L71

File: dashboard/components/modal/Modal.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+const meta: Meta<typeof Modal> = {
+  title: 'Komiser/Modal',
+  component: ModalWrapper,
+  decorators: [
+    Story => (
+      <div style={{ margin: '3em' }}>
+        {/* 👇 Decorators in Storybook also accept a function. Replace <Story/> with Story() to enable it  */}
+        {Story()}
```
Comment (Azanul) [inline dashboard/components/modal/Modal.stories.tsx:26]: The comment makes partial sense as `Story()` is used. I suggest either fix the comment or drop it altogether.

**Review by Azanul — CHANGES_REQUESTED** [review]
Could you also add comments over the SVGs in feedback widget indicating what they are

**Comment by Traxmaxx** [comment]
> Could you also add comments over the SVGs in feedback widget indicating what they are

I actually plan on moving them into `/icons` in the next steps and then also add all of them to storybook. What do you think? There should be a common place in storybook where contributors can look up all the icons existing already

**Review by mlabouardy — APPROVED** [review]

**Pull request merged**